content/en/agent/autodiscovery/ad_identifiers.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Autodiscovery Container Identifier
+kind: documentation
+disable_toc: true
+further_reading:
+- link: "/agent/autodiscovery/integrations"
+  tag: "Documentation"
+  text: "Create and load an Autodiscovery Integration Template"
+- link: "/agent/autodiscovery/management"
+  tag: "Documentation"
+  text: "Manage which Container to include in the Agent Autodiscovery"
+---
+
+Autodiscovery Container Identifiers or `ad_identifiers` are used when configuring the Agent Autodiscovery with files. The use case solved is to properly match the configuration file defined for an Autodiscovery Integration with a given container.
+
+The general format of an Autodiscovery file configuration is:
+
+```
+ad_identifier:
+  <INTEGRATION_AUTODISCOVEY_IDENTIFIER>
+
+init_config:
+  <INIT_CONFIG>
+
+instances:
+  <INSTANCES_CONFIG>
+```
+
+With the other types of configuration: Key-Value, Docker labels, or Kubernetes pod annotations, the matching between an integration configuration and the corresponding container is based on the `<CONTAINER_IDENTIFIER>` included in the labels/annotations, but with file configuration there is a need of properly matching the configuration file with the corresponding container.
+
+This can be achieved in two ways:
```

Comment:
What can be achieved? Matching the configuration file with the corresponding container?